From 830f0bab074f397bd13eec39fbef63b550a302b2 Mon Sep 17 00:00:00 2001
From: Richael Zhuang <richael.zhuang@arm.com>
Date: Thu, 28 Jan 2021 17:19:15 +0800
Subject: [PATCH 08/15] sock: add enable_quickack and enable_placement_id when
 writing json config

Signed-off-by: Richael Zhuang <richael.zhuang@arm.com>
Reviewed-on: https://review.spdk.io/gerrit/c/spdk/spdk/+/6144 (master)

(cherry picked from commit c0a3cf7b88851cf51dff8145546dd3c7e2741c9d)
Change-Id: If67dcb6bf7b9742d2c96481057b472e31e87e87d
Signed-off-by: Tomasz Zawadzki <tomasz.zawadzki@intel.com>
Reviewed-on: https://review.spdk.io/gerrit/c/spdk/spdk/+/6392
Tested-by: SPDK CI Jenkins <sys_sgci@intel.com>
Reviewed-by: Jim Harris <james.r.harris@intel.com>
Reviewed-by: Ben Walker <benjamin.walker@intel.com>
---
 lib/sock/sock.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/sock/sock.c b/lib/sock/sock.c
index ef4678f..8aa2817 100644
--- a/lib/sock/sock.c
+++ b/lib/sock/sock.c
@@ -832,6 +832,8 @@ spdk_sock_write_config_json(struct spdk_json_write_ctx *w)
 			spdk_json_write_named_uint32(w, "send_buf_size", opts.send_buf_size);
 			spdk_json_write_named_bool(w, "enable_recv_pipe", opts.enable_recv_pipe);
 			spdk_json_write_named_bool(w, "enable_zerocopy_send", opts.enable_zerocopy_send);
+			spdk_json_write_named_bool(w, "enable_quickack", opts.enable_quickack);
+			spdk_json_write_named_bool(w, "enable_placement_id", opts.enable_placement_id);
 			spdk_json_write_object_end(w);
 			spdk_json_write_object_end(w);
 		} else {
-- 
1.8.3.1

